src/atoms/Text/index.js

```diff
@@ -5,19 +5,77 @@ import classnames from 'classnames';
 import colors from 'atoms/Color/colors.module.scss';
 import styles from './text.module.scss';
 
+const convertChild = (child) => {
+  // Check children for other TextComponents
+  // and automatically convert them into <spans> if they are <p>.
+  if (child.type === Text && child.props.tag === 'p') {
+    return React.cloneElement(child, { tag: 'span' });
+  }
+
+  return child;
+};
+
+function Em(props) {
+  const newProps = Object.assign({}, props, {
+    tag: 'em',
+  });
+
+  return Text(newProps);
+}
+
+function Strong(props) {
+  const newProps = Object.assign({}, props, {
+    tag: 'strong',
```

Comment:
This will automatically bump the weight, I believe, and the `strong` weight isn't set in the CSS. Perhaps add it to `pg-normalize`?